uyuni-docs-helper

```diff
@@ -140,8 +146,7 @@ if [ -z ${LOCALCLONE} ]; then
     print_incorrect_syntax
   fi
   if [ -z ${GITREF} ]; then
-    # Default Git reference
-    GITREF='master'
+    print_incorrect_syntax
```

Comment:
Strictly speaking, this part of the code was right, but I moved the definition above.